### DISCUSSION TITLE
ERROR: Unsupported host operating system

### DISCUSSION THREAD

**Discussion by balajikulkarni** [zephyrproject-rtos/zephyr#50233] [Q&A]
Hi Team, 

I am trying to setup Zephyr SDK on my Ubuntu 18.04 machine (x86_64) following instructions from here
https://docs.zephyrproject.org/latest/develop/getting_started/installation_linux.html#install-requirements-and-dependencies

During SDK install & setup facing below error:

```
dev-station@dev-station-01:/local/mnt/workspace/misc/zephyr-sdk-0.15.0$ ./setup.sh
ERROR: Unsupported host operating system

dev-station@dev-station-01:/local/mnt/workspace/misc/zephyr-sdk-0.15.0$ echo $HOSTTYPE
x86_64-linux
```

Could you please share any pointers that can help in successful installation of Zephyr?

**Comment by cameled** [comment]
Check `OSTYPE` ENV on your shell.

In `setup.sh:188`, `linux-gnu*` is expected.
